<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StockCars</class>
 <widget class="QMainWindow" name="StockCars">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1099</width>
    <height>729</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>554</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1980</width>
    <height>1020</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StockCars</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>19</y>
      <width>571</width>
      <height>601</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QTableWidget" name="TableCar">
       <property name="maximumSize">
        <size>
         <width>1024</width>
         <height>16777215</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string>Название модели</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Количество</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Цена</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>id</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>10</y>
      <width>396</width>
      <height>614</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QFormLayout" name="formLayout_2">
       <item row="1" column="0">
        <widget class="QLabel" name="label_13">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Тип</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="CarType_gb">
         <property name="title">
          <string/>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="Truck_rb">
            <property name="text">
             <string>Грузовой</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="PassengerCar_rb">
            <property name="text">
             <string>Легковой</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Bus_rb">
            <property name="text">
             <string>Автобус</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="Trailer_rb">
            <property name="text">
             <string>Прицеп</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_12">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Название модели</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="ModelName">
           <property name="maxLength">
            <number>30</number>
           </property>
           <property name="clearButtonEnabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="CarsInfo">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Основные характеристики</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Объем двигателя</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="EngineVolume_sp">
            <property name="minimum">
             <double>0.500000000000000</double>
            </property>
            <property name="maximum">
             <double>30.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Вместимость</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="Capacity_sp">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>300</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Грузоподъемность</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="Carrying_sp">
            <property name="minimum">
             <number>100</number>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_17">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Максимальная скорость</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="MaxSpeed_sp">
            <property name="minimum">
             <number>60</number>
            </property>
            <property name="maximum">
             <number>320</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Разгон до 100 км/ч</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDoubleSpinBox" name="Acceleration_sp">
            <property name="minimum">
             <double>5.000000000000000</double>
            </property>
            <property name="maximum">
             <double>120.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_19">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Наличие системы климат-контроля</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Climat_flag">
            <property name="text">
             <string>В наличии</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label_20">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Количество</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Цена</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="Price_sp">
         <property name="minimum">
          <number>20000</number>
         </property>
         <property name="maximum">
          <number>3000000</number>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Дата последнего завоза</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QDateEdit" name="CurrentDate_sp"/>
       </item>
       <item row="0" column="1">
        <widget class="QSpinBox" name="Quantity_sp">
         <property name="maximum">
          <number>1000</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <item>
        <widget class="QPushButton" name="SaveBtn">
         <property name="text">
          <string>Сохранить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="AddButton">
         <property name="text">
          <string>Добавить запись</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="CancelBtn">
         <property name="text">
          <string>Отмена</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="DelBtn">
         <property name="text">
          <string>Удалить</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="EditBtn">
         <property name="text">
          <string>Редактировать</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1099</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="separator"/>
    <addaction name="action_SaveAs"/>
    <addaction name="action_Save"/>
    <addaction name="action_SaveDb"/>
    <addaction name="action_LoadDb"/>
    <addaction name="separator"/>
    <addaction name="action_CreateDb"/>
   </widget>
   <addaction name="menufile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_SaveAs">
   <property name="text">
    <string>Сохранить как</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>Сохранить </string>
   </property>
  </action>
  <action name="action_LoadDb">
   <property name="text">
    <string>Загрузить базу данных</string>
   </property>
  </action>
  <action name="action_SaveDb">
   <property name="text">
    <string>Сохранить базу данных</string>
   </property>
  </action>
  <action name="action_CreateDb">
   <property name="text">
    <string>Создать базу данных</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
